strongbox-cron/strongbox-cron-api/src/main/java/org/carlspring/strongbox/cron/config/ApplicationStartupCronTasksInitiator.java

```diff
@@ -1,21 +1,17 @@
 package org.carlspring.strongbox.cron.config;
 
-import org.carlspring.strongbox.cron.domain.CronTaskConfiguration;
-import org.carlspring.strongbox.cron.domain.CronTasksConfiguration;
+import org.carlspring.strongbox.cron.domain.CronTaskConfigurationDto;
+import org.carlspring.strongbox.cron.domain.CronTasksConfigurationDto;
```

Comment:
What's the difference between these two classes? (`CronTaskConfigurationDto` and `CronTasksConfigurationDto`)?

Reply:
plural `CronTasksConfigurationDto` contgains collection of singular `CronTaskConfigurationDto`